### PR TITLE
Correction of link and redirection to blog

### PR DESCRIPTION
**Pull request by joinimran**
There was confusion in the blog content at NSCI topic. I have removed the confusing part and updated the content. Additionally added the correct URL.

### CODE REVIEW

**Comment by jvsam** [comment]
Hi @nenonix and @JohanFreelancer9 for copy review please. Thank you. Refer: Issue #4003 and PR #4242

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ The following table lists management options for each setting, beginning with Wi
 | [11. Mail synchronization](#bkmk-mailsync) | ![Check mark](images/checkmark.png) | | ![Check mark](images/checkmark.png) |
 | [12. Microsoft Account](#bkmk-microsoft-account) | | ![Check mark](images/checkmark.png) |  ![Check mark](images/checkmark.png) |
 | [13. Microsoft Edge](#bkmk-edge) | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) |
-| [14. Network Connection Status Indicator](#bkmk-ncsi) | | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) |
+| [14. Network Connection Status Indicator](#bkmk-
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md]: Shouldn't it be .... (#bkmk-ncsi)?

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 2]
```diff
@@ -604,9 +605,9 @@ For a complete list of the Microsoft Edge policies, see [Available policies for
 
 ### <a href="" id="bkmk-ncsi"></a>14. Network Connection Status Indicator
 
-Network Connection Status Indicator (NCSI) detects Internet connectivity and corporate network connectivity status. NCSI sends a DNS request and HTTP query to http://www.msftconnecttest.com/connecttest.txt to determine if the device can communicate with the Internet. For more info about NCSI, see [The Network Connection Status Icon](https://techcommunity.microsoft.com/t5/Networking-Blog/bg-p/NetworkingBlog).
+Network Connection Status Indicator (NCSI) detects Internet connectivity and corporate network connectivity status. NCSI sends a DNS request and HTTP query to http://www.msftconnecttest.com/connecttest.txt to determine if the device can communicate with the Internet. [See Microsoft Networking Blog](https://techcommunity.microsoft.com/t5/Networking-Blog/bg-p/NetworkingBlog) to learn more about networking.
 
-In versions of Windows 10 prior to Windows 10, version 1607 and Windows Server 2016, the URL was `http://www.msftncsi.com`.
+In versions of Windows 10 prior to Windows 10, version 1607 and Windows Server 2016, the URL was `http://www.msftncsi.com/ncsi.txt`.
```
Comment (nenonix) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md]:
```suggestion
In versions of Windows 10 prior to version 1607 and Windows Server 2016, the URL was `http://www.msftncsi.com/ncsi.txt`.
```

**Comment by joinimran** [comment]
Suggested changes have been applied. Ready to Merge.

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 3]
```diff
@@ -604,9 +605,9 @@ For a complete list of the Microsoft Edge policies, see [Available policies for
 
 ### <a href="" id="bkmk-ncsi"></a>14. Network Connection Status Indicator
 
-Network Connection Status Indicator (NCSI) detects Internet connectivity and corporate network connectivity status. NCSI sends a DNS request and HTTP query to http://www.msftconnecttest.com/connecttest.txt to determine if the device can communicate with the Internet. For more info about NCSI, see [The Network Connection Status Icon](https://techcommunity.microsoft.com/t5/Networking-Blog/bg-p/NetworkingBlog).
+Network Connection Status Indicator (NCSI) detects Internet connectivity and corporate network connectivity status. NCSI sends a DNS request and HTTP query to http://www.msftconnecttest.com/connecttest.txt to determine if the device can communicate with the Internet. [See Microsoft Networking Blog](https://techcommunity.microsoft.com/t5/Networking-Blog/bg-p/NetworkingBlog) to learn more about networking.
```
Comment (nenonix) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md]:
```suggestion
Network Connection Status Indicator (NCSI) detects Internet connectivity and corporate network connectivity status. NCSI sends a DNS request and HTTP query to http://www.msftconnecttest.com/connecttest.txt to determine if the device can communicate with the Internet. See the [Microsoft Networking Blog](https://techcommunity.microsoft.com/t5/Networking-Blog/bg-p/NetworkingBlog) to learn more.
```

**Comment by joinimran** [comment]
Suggested change applied.

**Comment by jvsam** [comment]
Hi @Dansimp PR has been copy edited and is ready for your final review. Please see [files changed](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4298/files). Thank you very much.

**Comment by Dansimp** [comment]
@danihalfin can you take a look?

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 4]
```diff
@@ -69,7 +69,8 @@ The following table lists management options for each setting, beginning with Wi
 | [11. Mail synchronization](#bkmk-mailsync) | ![Check mark](images/checkmark.png) | | ![Check mark](images/checkmark.png) |
 | [12. Microsoft Account](#bkmk-microsoft-account) | | ![Check mark](images/checkmark.png) |  ![Check mark](images/checkmark.png) |
 | [13. Microsoft Edge](#bkmk-edge) | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) |
-| [14. Network Connection Status Indicator](#bkmk-ncsi) | | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) |
+| [14. Network Connection Status Indicator](#bkmk-ncsi)
+) | | ![Check mark](images/checkmark.png) | ![Check mark](images/checkmark.png) |
```
Comment (DaniHalfin) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md]: looks like this might be a formatting typo?

**Comment by DaniHalfin** [comment]
@medgarmedgar - can you take a look?

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md (hunk 4 above)
Comment (medgarmedgar) [reply windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md]: I guess I'm not seeing this one.  I see and extra ")" being added, but no corresponding "(".  Also, the bookmark links on the page are currently working.

**Comment by jvsam** [comment]
Hi @Dansimp is this good to merge since it has been approved by @medgarmedgar? This PR is related to the question I've raised [here](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4242) and since we cannot find the old article or anything close to it that provides more details, the suggestion is to just remove the link (including the texts).

image

Kindly merge if there are no issues. Thank you so much.

**Comment by jvsam** [comment]
Thank you!

**Comment by illfated** [comment]
As you can see, I have filed a PR for the issue noted above (https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4298#pullrequestreview-257597984).

**Pull request merged**